.github/workflows/release-comment-handler.yml

```diff
@@ -0,0 +1,148 @@
+name: Label Comment Handler
+
+on:
+  issues:
+    types:
+      - labeled
+
+jobs:
+  comment_on_label:
+    if: github.event.label.name == 'prerelease' || github.event.label.name == 'fixed'
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      contents: read
+    steps:
+      - name: Gather participants
+        id: participants
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const issue_number = context.payload.issue.number;
+            const repo = context.repo;
+
+            console.log(`Processing issue #${issue_number} in ${repo.owner}/${repo.repo}`);
+
+            // Get issue details (for author)
+            const issue = context.payload.issue;
+            const author = issue.user.login;
+            console.log(`Issue author: ${author}`);
+
+            // Use a Set to track all unique participants (normalized to lowercase for deduplication)
+            const all_participants = new Set();
+            
+            // Add the issue author
+            all_participants.add(author.toLowerCase());
+            console.log(`Added author to participants: ${author.toLowerCase()}`);
+
+            // Get all comments and add commenters
+            const comments = await github.paginate(
+              github.rest.issues.listComments,
+              { ...repo, issue_number }
+            );
+            console.log(`Found ${comments.length} comments`);
+            
+            comments.forEach(c => {
+              const commenter = c.user.login.toLowerCase();
+              all_participants.add(commenter);
+              console.log(`Added commenter: ${commenter}`);
+            });
+
+            // Get all reactions from comments
+            let totalCommentReactions = 0;
+            for (const comment of comments) {
```

Comment:
Loop through comments once  instead of twice but it doesn't really matter for github action.